lava-slave-install.md

```diff
@@ -0,0 +1,93 @@
+This is an howto install a lava-docker slave
+This howto is based on a Debian host.
+
+# Create a dedicated user
+The user will be lavadocker in our example. It must be in the docker group for doing docker operations.
+```
+useradd -G docker lavadocker
+```
+
+# Install all pre-requisites packages
+docker-compose need to be installed from PIP
+```
+apt-get install git python-pip
+```
+
+# Install docker-ce
+See https://docs.docker.com/install/linux/docker-ce/debian/ for more detailled informations.
+
+# install docker-compose for our user
+
+As lavadocker run:
+```
+pip install --user docker-compose
+```
+
+# Install a second network interface
+For our example it will be named enx0
+
+# Install a dhcpd listenning on enx0
+You have many choices, for our examples we will use isc-dhcp-server:
+```
+apt-get install isc-dhcp-server
+```
+
+## Configure the DHCPD server
+```
+sed -i 's,INTERFACESv4="",INTERFACESv4=""yourdedicatedinterfacename,' /etc/default/isc-dhcp-server
+```
+
+Add the following to /etc/dhcp/dhcpd.conf
+```
+subnet 192.168.66.0 netmask 255.255.255.0 {
```

Comment:
Is this 192.168.66.x subnet required because it's used elsewhere?  or is this just and example?
If it's different than 192.168.66.x, where else would this value need to be changed later?